packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx

```diff
@@ -26,13 +26,13 @@ Additionally, cloud sandboxes are collaborative by default, allowing for easier
 Besides the benefits presented above, there are some other noteworthy features that are only available in cloud sandboxes:
 - built-in [Docker support](/learn/environment/docker)
 - support for any programming language (using Docker)
+- Boxy [AI assistant](/learn/boxy/overview)
```

Comment:
good call to add this

Reply:
Does everything else look good to you, @necoline?